src/corecel/io/ScopedStreamFormat.hh

```diff
@@ -31,6 +31,13 @@ class ScopedStreamFormat
 
     // Restore formats on destruction
     inline ~ScopedStreamFormat();
+    //!@{
+    //! no move; no copying
```

Comment:
For consistency, please try to format comments like a sentence (but usually we omit the trailing period)
```suggestion
    //! Prevent moving and copying
```